src/hair/hair.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./hairStyle.css">
+  <link rel="stylesheet" href="reset.css">
+  <link href="https://fonts.googleapis.com/css?family=Josefin+Sans&display=swap" rel="stylesheet">
+  <title>Snazzy Hair Salon</title>
+</head>
+
+<body>
+  <aside>
+    <div class="logoimg">
+      <!-- with logo img as background in css -->
+    </div>
+    <!-- blank row -->
+    <nav id="navabout">
+      <h3>ABOUT</h3>
+    </nav>
+    <nav id="navmeet">
+      <h3>MEET</h3>
+    </nav>
+    <nav id="navservices">
+      <h3>SERVICES</h3>
+    </nav>
+    <nav id="navbootcamp">
+      <h3>BOOTCAMP</h3>
+    </nav>
+    <nav id="navproducts">
+      <h3>PRODUCTS</h3>
+    </nav>
+    <nav id="navlatest">
+      <h3>LATEST</h3>
+    </nav>
+    <nav id="navgiftcards">
+      <h3>GIFT CARDS</h3>
+    </nav>
+    <nav id="navcontact">
+      <h3>CONTACT</h3>
+    </nav>
+    <button id="buttonbook">BOOK NOW!</button>
+  </aside>
+
+  <header>
+    <h2 id="headerblurb">
+    "Here at Urban Betty we empower each other to grow. We remain connected with each other and our guests while constantly evolving."
+    </h2>
+  </header>
+
+  <main>
+    <p>
+      <h2>FIND THE PERFECT STYLIST FOR YOU</h2>
+      <h4>Use our specialized form to generate the best match for your personality & style.</h4>
+    </p>
+    <div>
+      <button class="buttonlocation" id="buttonlocation1">LOCATION 1</button>
+      <button class="buttonlocation" id="buttonlocation2">LOCATION 2</button>
```

Comment:
i don't know what the original image looks like but the button container is trying to swallow the text. lets add some padding!